pawno/include/PPC_PlayerCommands.inc

```diff
@@ -5145,3 +5248,52 @@ COMMAND:givelicense(playerid,params[])
 	// Let the server know that this was a valid command
 	return 1;
 }
+//This command let the admins to send an announce message
+COMMAND:announce(playerid, params[])
+{
+	// Check if the player has logged in
+	if (APlayerData[playerid][LoggedIn] == true)
+	{
+		// Check if the player's admin-level is at least 2
+		if(APlayerData[playerid][PlayerLevel] >= 2)
+		{
+			if(isnull(params)) 
+				return SendClientMessage(playerid, 0xFF0000AA, "Usage: \"/announce <message>\"");
+
+			GameTextForAll(params, 4000, 3);
+		}
+	}
+	return 1;
+}
+COMMAND:annn(playerid, params[]) return cmd_announce(playerid, params);
+
+// Let admins to refuel all player vehicles
+COMMAND:fuelall(playerid, params[])
+{
+	// Check if the player has logged in
+	if(APlayerData[playerid][LoggedIn] == true)
+	{
+		// Check if the player's admin-level is at least 2
+		if(APlayerData[playerid][PlayerLevel] >= 2)
+		{
+			// Setup local variables
+			new vModel;
+
+			// Loop through all vehicles
+			for(new vid = 1, j = GetVehiclePoolSize(); vid <= j; vid++)
```

Comment:
- Do not assign a number to `vid` as this will leave out a vehicle with id 0. Also use a semicolon instead of a comma.
- According to the wiki, `GetVehiclePoolSize` returns the highest playerid which is currently in use on the server. As we want to loop through all vehicles, we should use `MAX_VEHICLES`.
- Please stick to letter `i` instead of `j`.